app/component/CarLeg.js

```diff
@@ -59,8 +59,16 @@ function CarLeg(props, context) {
             defaultMessage="Drive {distance} ({duration})}"
           />
           {CarLeg.createBookButton(props.leg)}
-          <Icon img={carpoolAgencyIcon[0]} className="carpool-agency-logo" />
-          <Icon img={carpoolAgencyIcon[1]} className="carpool-agency-logo" />
+          <Icon
+            img={carpoolAgencyIcon[0]}
+            className="carpool-agency-logo"
+            tooltip={props.leg.route.agency.name}
+          />
+          <Icon
```

Comment:
What is rendered, if no second icon should be shown, like e.g. for Mifaz offers?

Reply:
Presumably an empty `<img>`. I think this could be avoided with a simple if statement or by putting the icons into a list and then mapping over them.

Reply:
Sadly yes, it is an empty img and the tooltip displays. I'll fix this.